libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -531,13 +531,13 @@ bool AP_BattMonitor::consumed_wh(float &wh, const uint8_t instance) const {
     }
 }
 
-/// capacity_remaining_pct - returns the % battery capacity remaining (0 ~ 100)
-uint8_t AP_BattMonitor::capacity_remaining_pct(uint8_t instance) const
+/// capacity_remaining_pct - returns true if the percentage is valid and writes to percentage argument
+bool AP_BattMonitor::capacity_remaining_pct(int8_t &percentage, uint8_t instance) const
 {
     if (instance < _num_instances && drivers[instance] != nullptr) {
-        return drivers[instance]->capacity_remaining_pct();
+        return drivers[instance]->capacity_remaining_pct(percentage);
     } else {
```

Comment:
pre-existing, don't need the `else` here

Reply:
done